README.md

```diff
@@ -35,30 +44,36 @@ Environment variables
 
 The following environment variables are exposed for InfraMock:
 
-`EXPERIMENT_ID` : set the experiment ID, or leave unset for default experiment.
-
-`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. This is
-set to `true` by default, which means that InfraMock will try to use a locally running version of
-the `api` to populate the localstack DynamoDB database with using its `/experiments/generate` endpoint.
-For this to work, `api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior.
+`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. 
```

Comment:
* Description of the default value is gone
* I would remove the reference to `PBMC` and just leave it as `mocked data set`

Reply:
* description of the default value is unchaged
* I've reworded the description as they don't need to be mock datasets at all

Reply:
> set to `true` by default

is gone. I think it is still relevant.